training/splitting_dataset.py

```diff
@@ -84,7 +84,9 @@ def get_random_split_results(
         return train_test_split(
             X, y, test_size=test_size, random_state=0, shuffle=True, stratify=y
         )
-    except ValueError:  # caused by setting stratify=y if there is a class in y that has only 1 member
+    except (
+        ValueError
```

Comment:
Dlaczego tu ten ValueError jest w nawiasie?

Reply:
Jestem prawie pewien, że `black` to tak przeformatował przez to, że komenatrz był w tej samej linii. Wystarczy przenieść komentarz do osobnej linii, wewnątrz tego errora, i będzie ok.

Inna sprawa - czemu w ogóle może zajść taka sytuacja? Po połączeniu klas powyżej poziomu 20 to jest możliwe? Bo to jest generalnie patologiczny case, wtedy trzeba coś z tym zrobić

Reply:
To ja tworzyłam ten kod i z tego co pamiętam błąd pojawiał się, kiedy jakiś (dowolny) level miał tylko jednego przedstawiciela. W takiej sytuacji połączenie klas >20 nic nie daje.